tests/python/direct/README.md

```diff
@@ -42,83 +42,89 @@ Each migrated test underwent the following adaptations:
 
 ### Tests Only in Main Frontend (Not in Direct)
 
-The following 77 tests exist in `tests/python/test_python_frontend.py` but are **NOT** present in `tests/python/direct/test_python_frontend.py`:
+The following 75 tests exist in `tests/python/test_python_frontend.py` but are **NOT** present in `tests/python/direct/test_python_frontend.py`:
```

Comment:
nitpick: having a number 75 is not going to be easily maintainable :)

Reply:
This section is supposed to be for the AI and will go away after the transition.